### PR TITLE
fix: colorpicker indicator is dissapearing when color is selected

### LINKED ISSUE #1655: fix: < ColorPicker /> indicator is dissapearing when color is selected
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/4315569094)

https://style.monday.com/?path=/docs/pickers-colorpicker--with-indicator

image
image

### PR DESCRIPTION
**Pull request by sayyedarib**
Fixes #1655 

**before**
Screenshot from 2023-10-12 19-26-12
Screenshot from 2023-10-12 19-26-00

**now**

#### Basic

- [x] Used plop (`npm run plop`) to create a new component.
- [x] PR has description.
- [x] New component is functional and uses Hooks.
- [x] Component is written in TypeScript.

#### Style

- [x] Styles are added to `NewComponent.modules.scss` file inside the `NewComponent` folder.
- [x] Component uses CSS Modules.
- [x] Design is compatible with [Monday Design System](https://design.monday.com/).
- [x] Styles are defined using [monday-ui-style](https://github.com/mondaycom/monday-ui-style) tokens
- [x] Displays correctly in all the themes

#### Storybook

- [x] Stories were added to `/src/components/NewComponent/__stories__/NewComponent.stories.mdx` file.
- [x] Stories include all flows of using the component.
- [x] Stories implemented using [vibe-storybook-components](https://github.com/mondaycom/vibe-storybook-components) components and tokens.

#### Tests

- [x] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

#### Accessibility

- [x] Component is accessible.
- [x] Component is compatible with [Vibe Accessibility Guidelines](https://style.monday.com/?path=/docs/foundations-accessibility--page)

### CODE REVIEW

File: src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.tsx [hunk 1]
```diff
@@ -37,7 +36,6 @@ const ColorPickerItemComponent: VibeComponent<ColorPickerItemComponentProps> = f
       colorStyle = ColorStyle.REGULAR,
       shouldRenderIndicatorWithoutBackground,
       ColorIndicatorIcon,
-      SelectedIndicatorIcon,
```
Comment (SergeyRoyt) [inline src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.tsx:40]: I think it just should have a default value of `ColorIndicatorIcon`

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
What you are proposing is a removing this feature completely - we can't do it, it's a breaking change
I believe that instead this `SelectedIndicatorIcon` if undefined should be equal to `SelectedIndicatorIcon`

**Comment by sayyedarib** [comment]
@SergeyRoyt let me know if the changes are ok ?

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Hey @sayyedarib!
The logic here is correct, but I believe it would work the same if you revert this line and just in props deconstruction give it default value and in terms of code cleanliness it'd look better
image

**Review by SergeyRoyt — APPROVED** [review]
Great job!  💪
Congrats with the first contribution to Vibe! 🥳🥳

**Pull request merged**